<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>442</width>
    <height>769</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>ocioLutPrescription</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="configTabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>421</width>
      <height>361</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="configTab">
     <attribute name="title">
      <string>Existing OCIO Config</string>
     </attribute>
     <widget class="QGroupBox" name="ocioGroupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>401</width>
        <height>111</height>
       </rect>
      </property>
      <property name="title">
       <string>OCIO Environment</string>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>381</width>
         <height>27</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QLabel" name="ocioCfgLabel">
          <property name="text">
           <string>OCIO</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="ocioCfgLineEdit">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="placeholderText">
           <string>&lt;Load a valid ocio configuration&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="ocioCfgLoadPushButton">
          <property name="text">
           <string>Load</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>13</x>
         <y>70</y>
         <width>372</width>
         <height>27</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <widget class="QLabel" name="ocioSeqLabel">
          <property name="text">
           <string>SEQ</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="ocioSeqLineEdit">
          <property name="placeholderText">
           <string>&lt;optional&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="ocioShotLabel">
          <property name="text">
           <string>SHOT</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="ocioShotLineEdit">
          <property name="placeholderText">
           <string>&lt;optional&gt;</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QGroupBox" name="lutOutputGroupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>210</y>
        <width>401</width>
        <height>101</height>
       </rect>
      </property>
      <property name="title">
       <string>LUT Output</string>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>381</width>
         <height>62</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QRadioButton" name="outputColorSpacesRadioButton">
            <property name="text">
             <string>Output ColorSpace</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="outputColorSpacesComboBox">
            <property name="editable">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QRadioButton" name="looksRadioButton">
            <property name="text">
             <string>Look</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="looksComboBox">
            <property name="editable">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>401</width>
        <height>27</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="inputColorSpacesLabel">
         <property name="text">
          <string>Input ColorSpace</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="inputColorSpacesComboBox">
         <property name="editable">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>170</y>
        <width>401</width>
        <height>27</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QCheckBox" name="shaperColorSpacesCheckBox">
         <property name="text">
          <string>Shaper ColorSpace</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="shaperColorSpacesComboBox">
         <property name="editable">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="configFreeTab">
     <attribute name="title">
      <string>Config-Free LUT Baking</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QPushButton" name="processBakeLutPushButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>660</y>
      <width>421</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Bake LUT</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="outputDirGroupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>570</y>
      <width>421</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>Output Directory</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>391</width>
       <height>27</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="outputDirLineEdit"/>
      </item>
      <item>
       <widget class="QPushButton" name="outputDirBrowsePushButton">
        <property name="text">
         <string>Browse</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QTabWidget" name="optionsTabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>380</y>
      <width>421</width>
      <height>181</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="bakingOptionsTab">
     <attribute name="title">
      <string>Baking Options</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>11</y>
        <width>388</width>
        <height>131</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="lutFormatLabel">
           <property name="text">
            <string>LUT Format</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="lutFormatComboBox"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QCheckBox" name="cubeSizeCheckBox">
           <property name="text">
            <string>Cube Size</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cubeSizeComboBox"/>
         </item>
         <item>
          <widget class="QCheckBox" name="shaperSizeCheckBox">
           <property name="text">
            <string>Shaper Size</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="shaperSizeComboBox"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="iccBakingOptionsTab">
     <attribute name="title">
      <string>ICC Reserved Options</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>401</width>
        <height>131</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <item>
          <widget class="QCheckBox" name="iccWhitePointCheckBox">
           <property name="text">
            <string>White Point</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="iccWhitePointLineEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_11">
         <item>
          <widget class="QCheckBox" name="iccDisplaysCheckBox">
           <property name="text">
            <string>Display</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="iccDisplaysComboBox">
           <property name="editable">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_12">
         <item>
          <widget class="QCheckBox" name="iccDescriptionCheckBox">
           <property name="text">
            <string>Description</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="iccDescriptionLineEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_13">
         <item>
          <widget class="QCheckBox" name="iccCopyrightCheckBox">
           <property name="text">
            <string>Copyright</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="iccCopyrightLineEdit"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>442</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>cubeSizeCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>cubeSizeComboBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>65</x>
     <y>541</y>
    </hint>
    <hint type="destinationlabel">
     <x>164</x>
     <y>540</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>shaperSizeCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>shaperSizeComboBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>276</x>
     <y>545</y>
    </hint>
    <hint type="destinationlabel">
     <x>386</x>
     <y>541</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>shaperColorSpacesCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>shaperColorSpacesComboBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>125</x>
     <y>248</y>
    </hint>
    <hint type="destinationlabel">
     <x>287</x>
     <y>244</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
